test/src/native_updater_test.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:native_updater/src/native_updater.dart';
+import 'package:in_app_update/in_app_update.dart';
+import 'package:native_updater/src/update_cupertino_alert.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  NativeUpdater nativeUpdater;
+  MethodChannel inAppUpdateChannel = const MethodChannel("in_app_update");
+  bool isImmediateUpdateInvoked;
+  bool isFlexibleUpdateInvoked;
+
+  setUp(() {
+    nativeUpdater = NativeUpdater();
+    isImmediateUpdateInvoked = false;
+    isFlexibleUpdateInvoked = false;
+
+    inAppUpdateChannel.setMockMethodCallHandler((MethodCall methodCall) async {
+      if (methodCall.method == 'checkForUpdate') {
+        return {
+          'updateAvailable': true,
+          'immediateAllowed': true,
+          'flexibleAllowed': true,
+          'availableVersionCode': 10,
+        };
+      }
+
+      if (methodCall.method == 'performImmediateUpdate') {
+        isImmediateUpdateInvoked = true;
+      }
+
+      if (methodCall.method == 'startFlexibleUpdate') {
+        isFlexibleUpdateInvoked = true;
+      }
+
+      return null;
+    });
+  });
+
+  group('Native Updater Test', () {
+    bool tForceUpdate;
+
+    group('Android', () {
+      test('Android platform should integrate In App Update', () async {
+        nativeUpdater.showMaterialAlertDialog();
+        find.byElementType(InAppUpdate);
+      });
+
+      test('Android platform should do immediate update if forceUpdate is true',
+          () async {
+        // arrange
```

Comment:
Please remove comments, your code should speak for itself.

Instead, add empty lines to create clear blocks, for instance:

```
        tForceUpdate = true;
        nativeUpdater.setForceUpdate(tForceUpdate);

        nativeUpdater.showMaterialAlertDialog();

        Future.delayed(Duration(seconds: 2), () {
          expect(isImmediateUpdateInvoked, true);
        });
```

Reply:
Done.

Reply:
Resolved on 876d9ef